app/assets/stylesheets/govuk_publishing_components/components/_checkbox.scss

```diff
@@ -1,2 +1,8 @@
-@import "helpers/govuk-frontend-settings";
-@import "govuk-frontend/components/checkboxes/checkboxes";
```

Comment:
This probably needs to be added in as the component might be stand alone

Reply:
Will that duplicate anything @DilwoarH ?

Reply:
My understanding is that when you use the gem in an app it pulls in all of the CSS from the gem, not just from the components that are used. Therefore as long as these imports are included somewhere, they'll get in. I think that perhaps (unless duplication doesn't occur) they should be moved from the `checkboxes` component into the `checkbox` component, instead.

Reply:
@DilwoarH have checked with @alex-ju and apparently if we include these lines twice there _will_ be duplication, which I want to avoid. I would suggest therefore that these lines go in only the checkbox component, since it's the 'original' component and checkboxes is using it. I'll add a separate commit to do this.